tests/index.js

```diff
@@ -50,10 +51,10 @@ describe('broccoli-gzip', function(){
 
     builder = new broccoli.Builder(tree);
     return builder.build().then(function(build) {
-      var gzippedText = fs.readFileSync(build.directory + '/test.txt.gz');
+      var gzippedText = fs.readFileSync(builder.outputPath + '/test.txt.gz');
       return RSVP.hash({
-        dir: build.directory,
-        actualCsv: fs.readFileSync(build.directory + '/test.csv'),
+        dir: builder.outputPath,
+        actualCsv: fs.readFileSync(builder.outputPath + '/test.csv'),
         actualText: RSVP.denodeify(zlib.gunzip)(gzippedText)
```

Comment:
Upgrading only Broccoli in #6 to get past our internal blockers but still interested in our the persistent filter is working here.